packages/styling/src/utilities/icons.ts

```diff
@@ -108,11 +119,47 @@ export function registerIcons(iconSubset: IIconSubset, options?: Partial<IIconOp
   }
 }
 
+/**
+ * Unregisters icons by name.
+ *
+ * @param iconNames - List of icons to unregister.
+ */
+export function unregisterIcons(iconNames: string[]): void {
+  const options = _iconSettings.__options;
+
+  for (const iconName of iconNames) {
```

Comment:
I personally prefer forEach syntax to for of, but that's just my preference

Reply:
the `of` syntax is slightly faster AFAIK. no closure required.